macaddress.go

```diff
@@ -0,0 +1,47 @@
+// vim:ts=2:sw=2:et:ai:sts=2
+package main
+
+import (
+	"errors"
+	"fmt"
+	"net"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+type macAddressValue struct{ v *net.HardwareAddr }
+
+func newMacAddressValue(p *net.HardwareAddr) *macAddressValue {
```

Comment:
This function isn't exported and only called once. Can it be inlined?

Reply:
As you can see, I really don't know how to code in Go :)
I did inline this, and as a result I also simplified the type.